spec/features/cart/increment_cart_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Cart inrementation' do
+    before(:each) do
+      @mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
```

Comment:
Story 23 Spec; This spec goes through and adds items and makes sure we are on the cart view before testing. We add the paper item until it is beyond what is in stock and make sure that it sends a flash message if you try to order too many